Add docs for Prestodb to MySQL type mapping in MySQL Connector (#21171)
prestodb/presto: presto-docs/src/main/sphinx/connector/mysql.rst

If you used a different name for your catalog properties file, use
that catalog name instead of ``mysql`` in the above examples.

Type mapping
------------

> Because PrestoDB and MySQL each support types that the other does not. When reading from or writing to MySQL,

Reviewer: ```suggestion
PrestoDB and MySQL each support types that the other does not. When reading from or writing to MySQL,
```
"Because" is not needed at the beginning of this sentence. If this sentence was the beginning of a longer sentence it would be appropriate here. "PrestoDB and MySQL each support types that the other does not." states the problem, and the rest of this section explains how the problem is addressed.

> Presto will convert the data types from MySQL to equivalent Presto data types and vice versa.

Reviewer: ```suggestion
Presto converts the data types from MySQL to equivalent Presto data types, and from Presto to equivalent MySQL data types.
```
Vice versa is a Latin phrase that should not be used in documentation. The [GitLab word list](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html) recommends "Do not use Latin abbreviations.", but does not have a specific entry or advice for avoiding Latin phrases such as vice versa. 

The [Google documentation style guide has an entry for vice versa](https://developers.google.com/style/word-list#vice-versa) and suggests "In such cases, make it explicitly clear what two things are swapped." 

My suggestion here attempts to be clear what two things are swapped. 
If I have changed the meaning in a way that you disagree with, I'm happy to discuss. 

Separate change: "will" is future tense and I'm suggesting a change to present tense, "Presto will convert" becomes "Presto converts".

Refer to the following sections for type mapping in each direction.

MySQL to PrestoDB type mapping
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The connector maps MySQL types to the corresponding PrestoDB types following

Reviewer: ```suggestion
The connector maps MySQL types to the corresponding PrestoDB types:
```
suggest deleting "following this table"

> this table:

Reviewer: ```suggestion
```
